### PR TITLE
fix jsdocs

### PR DESCRIPTION
**Pull request by maraoz**

### CODE REVIEW

File: lib/message.js [hunk 1]
```diff
@@ -16,9 +16,9 @@ var $ = bitcore.util.preconditions;
  * constructs a new message to sign and verify.
  *
  * @param {String} message
- * @returns {Message}
```
Comment (braydonf) [inline lib/message.js]: @returns {Message} is correct (so it's chainable)

Comment (eordano) [inline lib/message.js]: Hmm, that's right. In any case I thought that since this is a constructor it could be infered?

Comment (thofmann) [inline lib/message.js]: For reference, there isn't complete consistency for this in other classes. Address and PrivateKey both have the return documented, while Transaction and TransactionSignature do not.

Comment (braydonf) [inline lib/message.js]: Correct it can be inferred since it's a constructor (though often `new` isn't used, so it may be helpful to have this anyways).

**Pull request closed without merging**